### PR TITLE
Merge local-cluster-slow-[1,2] into one

### PR DESCRIPTION
**Pull request by brooksprumo**
#### Problem

Two local-cluster-slow jobs instead of one consumes an extra CI agent. Original context: https://github.com/solana-labs/solana/pull/24952#discussion_r865205232

#### Summary of Changes

Merge local-cluster-slow-[1,2] back into a single CI job.

### CODE REVIEW

File: ci/buildkite-pipeline.sh [hunk 1]
```diff
@@ -305,13 +305,9 @@ EOF
...
-  command_step "local-cluster-slow-1" \
-    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow-1.sh" \
-    40
-
-  command_step "local-cluster-slow-2" \
-    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow-2.sh" \
-    40
+  command_step "local-cluster-slow" \
+    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow.sh" \
+    80
```
Comment (brooksprumo) [inline ci/buildkite-pipeline.sh:310]: I bumped the timeout up to 80 mins here, but the CI for local-cluster-slow completed in ~10 mins (this time). What do you think is a good value here? Back to 40?

**Comment by CriesofCarrots** [comment]
It doesn't look like that step executed right, which is why it only took 10min:
```
++exec docker run --workdir /solana --volume /var/lib/buildkite-agent/builds/buildkite-agents-q7hv-1/solana-labs/solana:/solana --rm --volume /var/lib/buildkite-agent:/home --env HOME=/home --env CARGO_HOME=/home/.cargo --security-opt seccomp=unconfined --user 997:997 --env BUILDKITE --env BUILDKITE_AGENT_ACCESS_TOKEN --env BUILDKITE_JOB_ID --env CI --env CI_BRANCH --env CI_BASE_BRANCH --env CI_TAG --env CI_BUILD_ID --env CI_COMMIT --env CI_JOB_ID --env CI_PULL_REQUEST --env CI_REPO_SLUG --env CRATES_IO_TOKEN -e CODECOV_ENV -e CODECOV_TOKEN -e CODECOV_URL -e CODECOV_SLUG -e VCS_COMMIT_ID -e VCS_BRANCH_NAME -e VCS_PULL_REQUEST -e VCS_SLUG -e VCS_TAG -e CI_BUILD_URL -e CI_BUILD_ID -e CI_JOB_ID -e CI -e BUILDKITE -e BUILDKITE_BRANCH -e BUILDKITE_BUILD_NUMBER -e BUILDKITE_JOB_ID -e BUILDKITE_BUILD_URL -e BUILDKITE_PROJECT_SLUG -e BUILDKITE_COMMIT solanalabs/rust:1.60.0 ci/test-local-cluster-slow.sh
Executing test-local-cluster-slow
Error: Unknown test: test-local-cluster-slow
```

Also, I don't think we want to be running localnet-sanity on this job

**Comment by CriesofCarrots** [comment]
Both issues should be fixed with an update to test-stable.sh, about here: https://github.com/solana-labs/solana/blob/9bca909f6331743764146fead504aeb165c84a95/ci/test-stable.sh#L115

File: ci/buildkite-pipeline.sh (hunk 1 above)
Comment (brooksprumo) [reply ci/buildkite-pipeline.sh:310]: The passing run took ~36 mins. The two failing runs were 52 and 58 mins.

**Pull request closed without merging**